lib/logger.js

```diff
@@ -1,10 +1,10 @@
+/* eslint-disable no-magic-numbers */
 /**
  * Module dependencies.
  */
 
-var _ = require('lodash');
-var util = require('./util');
-var ut = require('util');
+const util = require('./util');
+const ut = require('util');
```

Comment:
There are linting rules but I believe they only work for `import/export`. I'll dig around and see if there are others.